src/control/SConscript

```diff
@@ -43,21 +43,12 @@ def is_firmware_mgmt_build(benv):
     "Check whether this build has firmware management enabled."
     return benv["FIRMWARE_MGMT"] == 1
 
-
-def is_ucx_build():
```

Comment:
I think it would still be possible for us to build without ucx (USE_INSTALLED and ucx is not installed). I'm not sure how much that would be used by developers though.

Reply:
yeah, we probably would need an "EXCLUDE" option for this.  I think we had one previously but removed it.  I'm not sure how important it is to have this capability.

Reply:
I think we could make the same arguments for libfabric.  With that in mind, this is at least consistent.   We can potentially explore adding the capability to exclude.